resources/assets/lib/models/chat/message.ts

```diff
@@ -39,6 +41,7 @@ export default class Message {
     return Object.assign(message, {
       channelId: json.channel_id,
       content: json.content,
+      contentHtml: json.content_html,
```

Comment:
not related but is there specific reason using assign instead of just usual assignment (in a `runInAction`)?

`Object.assign` isn't type checked.

Reply:
it is a mystery 🤔

Reply:
it only seems to avoid `message.` on every line and isn't even less lines 🤔